[2023-04-26T01:03:42.574+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: email_processing.send_email scheduled__2023-04-25T00:00:00+00:00 [queued]>
[2023-04-26T01:03:42.592+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: email_processing.send_email scheduled__2023-04-25T00:00:00+00:00 [queued]>
[2023-04-26T01:03:42.593+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T01:03:42.595+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-04-26T01:03:42.595+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T01:03:42.623+0000] {taskinstance.py:1300} INFO - Executing <Task(EmailOperator): send_email> on 2023-04-25 00:00:00+00:00
[2023-04-26T01:03:42.639+0000] {standard_task_runner.py:55} INFO - Started process 5179 to run task
[2023-04-26T01:03:42.644+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'email_processing', 'send_email', 'scheduled__2023-04-25T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/email_processing.py', '--cfg-path', '/tmp/tmpnwur0h4r']
[2023-04-26T01:03:42.647+0000] {standard_task_runner.py:83} INFO - Job 32: Subtask send_email
[2023-04-26T01:03:42.776+0000] {task_command.py:388} INFO - Running <TaskInstance: email_processing.send_email scheduled__2023-04-25T00:00:00+00:00 [running]> on host 182d53144720
[2023-04-26T01:03:42.885+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=email_processing
AIRFLOW_CTX_TASK_ID=send_email
AIRFLOW_CTX_EXECUTION_DATE=2023-04-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-25T00:00:00+00:00
[2023-04-26T01:03:42.911+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/email.py:152: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2023-04-26T01:03:42.912+0000] {email.py:268} INFO - Email alerting: attempt 1
[2023-04-26T01:03:44.144+0000] {email.py:280} INFO - Sent an alert email to ['rahulkumar@sigmoidanalytics.com']
[2023-04-26T01:03:46.020+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=email_processing, task_id=send_email, execution_date=20230425T000000, start_date=20230426T010342, end_date=20230426T010346
[2023-04-26T01:03:46.069+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-04-26T01:03:46.083+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
